resolv.cabal

```diff
@@ -99,7 +99,7 @@ test-suite resolv.
                , bytestring
 
   -- additional dependencies not inherited
-  build-depends: tasty         >= 0.11.2  && < 0.12
-               , tasty-hunit   >= 0.9.2   && < 0.10
+  build-depends: tasty         >= 0.11.2  && < 1.2
```

Comment:
if possible, I only want to support a single major version of test-dependency

Reply:
Done.